### PR TITLE
Handle string types in compatible way

### PR DESCRIPTION
**Pull request by shaded-enmity**
i.e. make sure that we can pass `unicode` in Py2 context

### CODE REVIEW

**Comment by pirat89** [comment]
@shaded-enmity It looks that at least these lines should be fixed as well:
```
leapp/utils/audit/__init__.py:        if not isinstance(configuration, str):
leapp/libraries/stdlib/call.py:    elif isinstance(stdin, str):
```

File: leapp/reporting/__init__.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from leapp.models import fields, Model
 from leapp.topics import ReportTopic
 from leapp.libraries.stdlib.api import produce
+from leapp.compat import string_types
```
Comment (fernflower) [inline leapp/reporting/__init__.py]: Please rearrange imports in alphabetical order. 

Hmm, wondering why didn't linter catch that?

**Comment by fernflower** [comment]
leapp-ci build

**Comment by Rezney** [comment]
Looks good to me. Running some tests...

**Pull request merged**